.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
```

Comment:
For this file to take effect, dependabot needs to be enabled for the repo. @chaoran-chen, is it enabled? https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuring-dependabot-version-updates#enabling-dependabot-version-updates